client_app/src/components/volunteer/ShiftsData.js

```diff
@@ -0,0 +1,42 @@
+const ShiftsData = {
+  "shelter-1": {
```

Comment:
This should be a shelter id, for example 6093. I bet this would fix your test problem.